casper/src/main/scala/coop/rchain/casper/genesis/contracts/ProofOfStake.scala

```diff
@@ -14,7 +14,8 @@ final case class ProofOfStake(
     maximumBond: Long,
     validators: Seq[Validator],
     epochLength: Int,
-    quarantineLength: Int
+    quarantineLength: Int,
+    numberOfActiveValidators: Int = 100
```

Comment:
It is better to put the `numberOfActiveValidators` in th configuration.

Reply:
Which configuration?